unit_tests/UnitTestNgpMesh1.C

```diff
@@ -181,3 +182,177 @@ TEST_F(Hex8MeshWithNSOFields, NGPMeshField)
 
   test_ngp_mesh_field_values(*bulk, velocity, massFlowRate);
 }
+
+//struct MyBaseClass
```

Comment:
I think it would be good to replace these stale comments with a brief comment about the nuances of why these tests were added.  They are all fresh in our minds now, but 2-3 years from now it might be hard to remember why we wanted/needed  to test a fake field generation for nalu-wind.

Reply:
Thanks @psakievich , I added a commit in which I ran clang-format and addressed your comments cleaning up the new unit-test.